btap/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#FROM python:3.7
+# FROM python:3.8-slim
+FROM ubuntu:20.04
+
+RUN apt-get -y update && apt-get install -y python3 python3-pip

```

Comment:
@sampasha See the note here about removing the apt-cache after you're done installing

https://runnable.com/blog/9-common-dockerfile-mistakes

Reply:
@blairdrummond This is resolved, please check.